### PR TITLE
FIX display images in readme, IMP _extract_data_and_upd_supplierinfo()

### PR DESCRIPTION
**Pull request by bealdav**

### CODE REVIEW

File: supplier_inventory_import_ubl/tests/test_import.py [hunk 1]
```diff
@@ -27,17 +27,22 @@ def test_only_one_default_code(self):
         self.assertEqual(info, False)
         self.assertEqual(
             res.get("res_model"),
-            "product.supplierinfo",
+            "product.product",
             "Current model should be product.supplierinfo",
```
Comment (sebastienbeau) [inline supplier_inventory_import_ubl/tests/test_import.py]: wrong message

File: supplier_inventory_import_ubl/tests/test_import.py [hunk 2]
```diff
@@ -27,17 +27,22 @@ def test_only_one_default_code(self):
...
             "no_seller_reference"
         )
-        wizard.process_document()
+        res = wizard.process_document()
         info = self.env.ref("product.product_product_35").supplier_stock_info
         self.assertEqual(info and info[:4], "Wood")
+        self.assertEqual(
+            res.get("res_model"),
+            "product.product",
+            "Current model should be product.supplierinfo",
```
Comment (sebastienbeau) [inline supplier_inventory_import_ubl/tests/test_import.py]: wrong message

Comment (bealdav) [reply supplier_inventory_import_ubl/tests/test_import.py]: Caramba !

**Review by sebastienbeau — APPROVED** [review]
LGTM (code review)

**Pull request merged**